applications/nrf_desktop/configuration/nrf54h20dk_nrf54h20_cpuapp/prj.conf

```diff
@@ -0,0 +1,116 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+################################################################################
+# Application Configuration
+
+CONFIG_DESKTOP_INIT_LOG_MOTION_EVENT=n
+CONFIG_DESKTOP_INIT_LOG_HID_REPORT_EVENT=n
+CONFIG_DESKTOP_INIT_LOG_HID_REPORT_SENT_EVENT=n
+CONFIG_CAF_INIT_LOG_KEEP_ALIVE_EVENTS=n
+
+CONFIG_DESKTOP_ROLE_HID_PERIPHERAL=y
+CONFIG_DESKTOP_PERIPHERAL_TYPE_MOUSE=y
+CONFIG_DESKTOP_DEVICE_PID=0x52DE
+CONFIG_DESKTOP_HID_BOOT_INTERFACE_MOUSE=y
+
+CONFIG_DESKTOP_MOTION_BUTTONS_ENABLE=y
+CONFIG_DESKTOP_MOTION_BUTTONS_UP_KEY_ID=0
+CONFIG_DESKTOP_MOTION_BUTTONS_DOWN_KEY_ID=3
+CONFIG_DESKTOP_MOTION_BUTTONS_LEFT_KEY_ID=2
+CONFIG_DESKTOP_MOTION_BUTTONS_RIGHT_KEY_ID=1
+
+CONFIG_CAF_BUTTONS=y
+CONFIG_CAF_BUTTONS_POLARITY_INVERSED=y
+CONFIG_CAF_BUTTONS_PM_KEEP_ALIVE=n
+
+CONFIG_CAF_CLICK_DETECTOR=y
+
+CONFIG_CAF_LEDS=y
+
+CONFIG_CAF_FORCE_POWER_DOWN_EVENTS=y
```

Comment:
It fails build without it

Reply:
I guess, we refer to the event without `IS_ENABLED(CONFIG_CAF_FORCE_POWER_DOWN_EVENTS)`. Then I would suggest to add the missing `IS_ENABLED(CONFIG_CAF_FORCE_POWER_DOWN_EVENTS)`, instead of working it around in the configuration (should be really simple to fix it properly in code).